internal/generate/olm-catalog/csv_cmd_prompt.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olmcatalog
+
+import (
+	"strings"
+
+	olmapiv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+)
+
+// InteractiveCSVCmd includes the list of CSV fields which would be asked
+// to the user while generating CSV.
+type InteractiveCSVCmd struct {
```

Comment:
No need to export this now.
```suggestion
type interactiveCSVCmd struct {
```